apps/factory/src/pages/dashboard/prompts/[id]/logs/index.tsx

```diff
@@ -163,8 +168,17 @@ const PromptLogTable: NextPageWithLayout = () => {
                     whiteSpace: "nowrap",
                   }}
                 >
-                  {log.completion}
-                  <p>tokens: {log.completion_tokens}</p>
+                  {log.llmModelType === ModelTypeSchema.Enum.TEXT2TEXT ? (
+                    <>
+                      {log.completion}
+                      <p>tokens: {log.completion_tokens}</p>
+                    </>
+                  ) : (
+                    <img
+                      className="h-48 w-96 object-contain"
+                      src={log.completion}
+                    />
+                  )}
```

Comment:
@pulkit21 see if we can convert this in to a component so we can reuse it. It seems this already have 2 usecases already.